cms/static/js/views/pages/container.js

```diff
@@ -191,6 +192,17 @@ define(['jquery', 'underscore', 'gettext', 'js/views/pages/base_page', 'common/j
                 this.duplicateComponent(this.findXBlockElement(event.target));
             },
 
+            moveXBlock: function(event) {
+                var xblockElement = this.findXBlockElement(event.target),
+                    modal = new MoveXBlockModal({
+                        sourceXBlockInfo: XBlockUtils.findXBlockInfo(xblockElement, this.model),
+                        XBlockUrlRoot: this.getURLRoot()
+                    });
+
+                event.preventDefault();
+                modal.show();
```

Comment:
@muhammad-ammar when /or how the concise course outline data request would be sent in this process ? just a curious question.